lib/DDG/Spice/Mtg.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Mtg;
+
+# mtg <card name> Returns infos on the Magic The Gathering Card
+
+use DDG::Spice;
+
+name "Mtg Card Search";
+description "mtg <card name> Returns infos on the Magic The Gathering Card";
+primary_example_queries "mtg nullify", "mtg Boros Reckoner";
+
+category "entertainment";
+topics "entertainment", "gaming";
+code_url "https://github.com/puskin94/zeroclickinfo-spice/blob/mtg/lib/DDG/Spice/Mtg.pm";
+attribution github => ["https://github.com/puskin94", "puskin"];
+
+triggers start => 'mtg';
```

Comment:
Maybe it should trigger with "magic" or "magic the gathering" or "magic cards" as well?